src/main/java/bio/terra/cli/service/utils/SamService.java

```diff
@@ -350,15 +348,12 @@ public void removeUserFromGroup(String groupName, GroupPolicy policy, String use
    * "/api/resources/v1/{resourceTypeName}/{resourceId}/policies/{policyName}/memberEmails/{email}"
    * PUT endpoint to add an email address to a resource + policy.
    *
-   * <p>If that returns a Bad Request error, then call the SAM "/api/users/v1/invite/{inviteeEmail}"
-   * endpoint to invite the user.
-   *
    * @param resourceType type of resource
    * @param resourceId id of resource
    * @param resourcePolicyName name of resource policy
    * @param userEmail email of the user or group to add
    */
-  public void addUserToResourceOrInviteUser(
+  public void addUserToResource(
```

Comment:
Changed to private. I think it's fine to call this from outside this class, but since there's no use case for that now, the visibility should be as restricted as possible.